src/main/java/org/sagebionetworks/bridge/dynamodb/DynamoDataPipelineHelper.java

```diff
@@ -0,0 +1,178 @@
+package org.sagebionetworks.bridge.dynamodb;
+
+import com.amazonaws.regions.Region;
+import com.amazonaws.services.datapipeline.model.Field;
+import com.amazonaws.services.datapipeline.model.PipelineObject;
+import com.amazonaws.services.dynamodbv2.model.TableDescription;
+import com.google.common.collect.Lists;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.ChronoUnit;
+import java.util.Collection;
+import java.util.List;
+
+/**
+ * Creates a Data Pipeline for backing up Dynamo DB tables within a region.
+ */
+public class DynamoDataPipelineHelper {
+
+    private static final String READ_THROUGHPUT_PERCENTAGE = "0.3";
```

Comment:
Based on the analysis in https://sagebionetworks.jira.com/browse/BRIDGE-1085 to maximize savings, we should make this configurable. The ratios range from 0.2 to 0.8.